tox.ini

```diff
@@ -36,6 +36,6 @@ deps=
 commands=flake8 security testing
 
 [flake8]
-ignore=E131,H306,H301,H404,H405,H101,N802
+ignore=E131,H306,H301,H404,H405,H101,N802,N812
```

Comment:
Where are we breaking N812?

Reply:
It doesn't like it when we import things as `_`